gh-actions-scripts/install_helm.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# download
+curl https://get.helm.sh/helm-v3.2.4-linux-amd64.tar.gz > helm.tar.gz
+tar -zxvf helm.tar.gz
+if [ ! -d "$GITHUB_WORKSPACE/bin" ]; then
```

Comment:
I guess I can answere it myself: the script is different, as we are using `$GITHUB_WORKSPACE` here.